### ISSUE TITLE
Add support for `array` to "typeorm-typescript/enforce-column-types" 

### ISSUE THREAD

**Issue by SimonSimCity** [daniel7grant/eslint-plugin-typeorm-typescript#3]
The rule currently reports at a property defined as:
```ts
@Entity('data_points')
export class DataPoint {
  @Column({ type: 'text', name: 'linked_persons', nullable: true, array: true })
  linkedPersons?: string[] | null;
}
```

... and provides the option to change the type of `linkedPerson` to `string | null`, which is wrong, because I have the option `array` enabled. See https://typeorm.io/entities#column-options

It should not report on this column. It should instead report on this property, where the property `array` is enabled, but the type is no array:
```ts
@Entity('data_points')
export class DataPoint {
  @Column({ type: 'text', name: 'linked_persons', nullable: true, array: true })
  linkedPersons?: string | null;
}
```

**Comment by daniel7grant** [comment]
Thanks for using the library! I'm not a TypeORM power user, so I don't know all of the `@Column` options. I'll add an extra check for the `array` field, to handle this case.

**Comment by daniel7grant** [comment]
I added support for the array type in [v0.2.6](https://github.com/daniel7grant/eslint-plugin-typeorm-typescript/releases/tag/v0.2.6). Can you try if this solves your problem?

**Comment by abd-almofleh** [comment]
@daniel7grant Yes it does, Thanks for the quick update

**Comment by SimonSimCity** [comment]
I can confirm this working.

I can fully understand that you built the library for what was needed at the time, and the more appreciate you taking the time to help extending it for my use-case. Thanks very much for the quick update 🎉
